apps/api/src/domain/domainModelValidators/__tests__/domainModelValidators/buildDomainModelValidatorTestCases/bibliographicReferenceInvalidTestCases/bookBibliographicReference.domainModelValidatorTestCases.ts

```diff
@@ -0,0 +1,138 @@
+        {
+            description: 'The title is an empty string',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    title: '',
+                } as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The creators array is empty',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    creators: [],
+                    // TODO remove cast
+                } as unknown as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The abstract is an empty string',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    abstract: '',
+                    // TODO remove cast
+                } as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The year is too big',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    year: 2100,
+                    // TODO remove cast
+                } as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The publisher is an empty string',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    publisher: '',
+                    // TODO remove cast
+                } as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The place is an empty string',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    place: '',
+                    // TODO remove cast
+                } as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The url is an invalid URL',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    url: '',
+                    // TODO remove cast
+                } as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The ISBN is an invalid ISBN',
```

Comment:
Be more specific as to how it is invalid. For example
``` 
5, [1,2,3], '', null, undefined, and ['12867584']
```
are all invalid values for an ISBN.